source/index.html

```diff
@@ -1,12 +1,167 @@
+        <div class="main-nav__wrapper">
+          <ul class="main-nav__list">
+            <li class="main-nav__item main-nav__item--current">
+              <a href="index.html">Главная</a>
+            </li>
+            <li class="main-nav__item">
+              <a href="catalog.html">Каталог продукции</a>
+            </li>
+            <li class="main-nav__item">
+              <a href="form.html">Подбор программы</a>
+            </li>
+          </ul>
+        </div>
+      </nav>
+    </header>
 
+    <main class="main-container">
+      <div class="hero">
+        <h1 class="main-title">Функциональное питание для котов</h1>
+        <span class="hero__slogan">Занялся собой? Займись котом!</span>
+        <img class="hero__image" src="img/can-hero.png" width="280" height="270" alt="product">
+        <a class="primary-button hero__button" href="form.html">Подобрать программу</a>
+      </div>
+
+      <div class="program-wrapper">
+        <h2 class="visually-hidden">Наши программы</h2>
+
+        <section class="program">
+          <h3 class="program__title program__title--slim">Похудение</h3>
+          <p class="program__descr">
+            Ваш кот весит больше собаки и почти утратил способность лазить
+            по деревьям? Пора на диету! Cat Energy Slim поможет вашему
+            питомцу сбросить лишний вес.
+          </p>
+          <a class="program__link" href="catalog.html">Каталог Slim</a>
+        </section>
+
+        <section class="program">
+          <h3 class="program__title program__title--pro">Набор массы</h3>
+          <p class="program__descr">
+            Заработать авторитет среди дворовых котов и даже собак?
+            Серия Cat Energy Pro поможет вашему коту нарастить необходимые мышцы!
+          </p>
+          <a class="program__link" href="catalog.html">Каталог Pro</a>
+        </section>
+      </div>
+
+      <div class="about">
+        <h2 class="secondary-title">Как это работает</h2>
+        <ul class="about__list">
+          <li class="about-item">
+            <p class="about-item__text">
+              Функциональное питание содержит только полезные питательные вещества.
+            </p>
+          </li>
+
+          <li class="about-item">
+            <p class="about-item__text">
+              Выпускается в виде порошка, который нужно лишь залить кипятком и готово.
+            </p>
+          </li>
+
+          <li class="about-item">
+            <p class="about-item__text">
+              Замените один-два приема обычной еды на наше функциональное питание.
+            </p>
+          </li>
+
+          <li class="about-item">
+            <p >
+              Уже через месяц наслаждайтесь изменениями к лучшему вашего питомца!
+            </p>
+          </li>
+        </ul>
+      </div>
+
+      <div class="example">
+        <h2 class="secondary-title">Живой пример</h2>
```

Comment:
Пока это не проблема, всегда можно добавить будет, но наверняка будет удобно `secondary-title` миксовать с классом родительского блока:
```suggestion
        <h2 class="secondary-title example__title">Живой пример</h2>
```

Reply:
👍